torch/_refs/__init__.py

```diff
@@ -2426,14 +2490,186 @@ def _normalize(
         out (Tensor): normalized tensor.
         mean (Tensor): mean of the tensor along norm_dims.
         rstd (Tensor): 1/std of the tensor along norm_dims.
+        unbiased_var (Tensor): variance of the tensor along norm_dims.
     """
     computation_dtype = utils.get_computation_dtype(a.dtype)
     a_acc = _maybe_convert_to_dtype(a, computation_dtype)
     assert isinstance(a_acc, TensorLike)  # to avoid mypy error for var_mean
-    biased_var, mean = var_mean(a_acc, dim=norm_dims, unbiased=False, keepdim=True)
-    rstd = torch.rsqrt(biased_var + eps)
+    unbiased_var, mean = var_mean(a_acc, dim=norm_dims, unbiased=False, keepdim=True)
```

Comment:
`unbiased` is set to `False` in the call, so renaming `biased_var` -> `unbiased_var` is incorrect